fastly/block_fastly_service_v1_splunk.go

```diff
@@ -69,6 +69,12 @@ func (h *SplunkServiceAttributeHandler) GetSchema() *schema.Schema {
 			Description: "The client private key used to make authenticated requests. Must be in PEM format.",
 			Sensitive:   true,
 		},
+		"use_tls": {
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Default:     false,
+			Description: "Whether to use TLS for secure logging. Defult: `false`",
```

Comment:
@smaeda-ks can you regenerate the docs please once applying this edit...

```suggestion
			Description: "Whether to use TLS for secure logging. Default: `false`",
```

Reply:
@Integralist Thank you. Fixed the typo.